src/components/atoms/ChatMessage/ChatMessage.tsx

```diff
@@ -75,13 +75,13 @@ export const ChatMessage: React.FC<ChatProps> = ({
             <RenderMarkdown text={text} allowHeadings={false} />
           </div>
 
-          <div className="ChatMessage__reply-icon">
+          <button className="ChatMessage__reply-icon">
             <FontAwesomeIcon
               icon={faReply}
               size="sm"
               onClick={selectThisThread}
             />
-          </div>
+          </button>
```

Comment:
On other buttons like this you've added `aria-label` and similar. Why not here? Can we please be consistent in the changes we're making so that we ensure they are equally applied in all locations.